src/tensors/cpu/wasm_intgemm_fallback.cpp

```diff
@@ -59,18 +59,22 @@ extern "C" void int8PrepareBFromQuantizedTransposedFallback(const int8_t* input_
 }
 
 extern "C" void int8PrepareBiasFallback(const int8_t* input_B_prepared,
-                                        float scale,
-                                        float zero_point,
+                                        float scale_A,
+                                        float zero_point_A,
+                                        float scale_B,
+                                        float zero_point_B,
+                                        float unquant_multiplier,
                                         Index width,
                                         Index cols_B,
                                         const float* input_bias,
                                         float* output) {
   LOG(info, "Calling fallback implementation of \"int8PrepareBias\"");
+  float unquant_factor = (-1) * ((127.0f / scale_A) * (127.0f / scale_B)) / (127.0f);
```

Comment:
I think this should be `unquant_multiplier *= `?

Reply:
You are right. I removed `unquant_multiplier` from prepareBias API altogether as per your recommendation 👍🏾